pii_scan.py

```diff
@@ -113,3 +113,12 @@ def analyze_text(text: str, show_supported=False, show_details=False, score_thre
     print(show_aggie_pride())
     print('Displaying supported entities')
     pp.pprint(analyze_text('This is a test', show_supported=True))
+
```

Comment:
Normally you want to be careful about unintentional whitespace changes like this one.  It can cause unintentional merge conflicts later on.  It's always better to revert a file back to its original state so this does not happen